src/EnergyPlus/FluidProperties.cc

```diff
@@ -102,29 +102,9 @@ namespace FluidProperties {
     // supplying the same data for concentrations of 0.0 and 1.0 only.
     // Temperature data has to be supplied in ascending order only.
 
-    // Data
-    // MODULE PARAMETER DEFINITIONS
-
-    // DERIVED TYPE DEFINITIONS
-
-    // INTERFACE BLOCK SPECIFICATIONS
-    // na
-
-    // MODULE VARIABLE DECLARATIONS
-
-    // ACCESSIBLE SPECIFICATIONS OF MODULE SUBROUTINES OR FUNCTONS:
-
-    // Object Data
-
```

Comment:
We initially wanted to have the cached version inlined, which required us to have a bunch of things in the `.hh` that wouldn't need to be there otherwise. The fact that the cache array is a global variable is just oversight I am guessing. This code predates the `state` idiom by a good bit.

Reply:
So is it worthwhile to move the inline function to the cc file?  I'm not sure how much we are gaining by having it marked as inline here.  Maybe a lot?  I could pull it into the cc and do a real-world experiment?

Reply:
I doubt it's "a lot" in C++/x64. I think it was more in Fortran/x86 and its heavier calling convention. We can probably reduce it more with a better API than with inlining at this point.

Reply:
What do you mean by "19 threads"? Are you using the API to run 19 separate simulations out of the same process? Or something else?

Reply:
Should've been a bit more precise: running the regression tool using 19 individual child processes.  Technically each with their own thread, but even more, their own process.  This is just how the regression tool works.

My main point was just that my "experiment" was to run both the main branch and the mod branch with lots of threads.  I did not try to do an isolated experiment where nothing else was running on the machine and they were run on a single thread, because time.  Anyway, the runtime basically did not change, so I think this change is definitely worth the removal of the global.